spec/migrations/20210519204247_move_miq_set_memberships_spec.rb

```diff
@@ -0,0 +1,77 @@
+require_migration
+
+describe MoveMiqSetMemberships do
+  let(:rel_stub)        { migration_stub(:Relationship) }
+  let(:membership_stub) { migration_stub(:MiqSetMembership) }
+  let(:ns_stub)         { Class.new(ActiveRecord::Base) { self.table_name = "miq_ae_namespaces" } }
+  let(:rp_stub)         { Class.new(ActiveRecord::Base) { self.table_name = "resource_pools" } }
+
+  migration_context :up do
+    context "two sets with one member each" do
```

Comment:
In the previous PR I had asked Drew if we could have specs for more complicated examples, such as empty sets and sets with multiple children.  Can we add them here as well?

Reply:
I will look for those comments and see if you have any other suggestions as well.  Since I was re-writing the `:down`, the first pass was to make sure what existed still worked, but I can add stuff now since we have a working copy.

Reply:
I can't find the comments, but I do recall saying them...it may have happened in gitter or gchat

Reply:
Yeah, I am just doing the following currently (haven't pushed the changes yet):

- 1 empty set
- 1 set with 1 child
- 1 set with 4 children, sharing one with the second

Reply:
oh good idea on the sharing...i forgot about that.